R/app-server.R

```diff
@@ -94,16 +95,45 @@ app_server <- function(input, output, session) {
         upload_documents_server,
         "documentation",
         parent_folder = reactive(created_folder),
-        study_table_id = reactive(config::get("study_table")),
+        study_names = all_studies,
         synapseclient = synapse,
         syn = syn
       )
     }
 
+    ## Reset fileInputs if reset button pressed
+    ## Need to somehow reset the study name?
```

Comment:
```suggestion
    ## Reset fileInputs, study name, and other inputs
```

Suggest tweaking the wording of this since the study name reset is working